### PR TITLE
Add Ubuntu Pro Info monitor

### PR DESCRIPTION
**Pull request by Perfect5th**

### CODE REVIEW

**Comment by silverdrake11** [comment]
This looks good. The only thing I noticed is the run_interval is missing. It's using the default I think which is too frequent. Should be like once every 60 or 15 minutes.

**Comment by Perfect5th** [comment]
I think 15 minutes makes sense. It's pretty CPU-light, so more frequent than 60 minutes is ok.

As for that bug... I'm not overly concerned. I doubt it'll happen often, and re-registering isn't the end of the world. As you said, it won't matter once the full pro changes are in place.

File: landscape/client/monitor/ubuntuproinfo.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    Plugin that captures and reports Ubuntu Pro registration
+    information.
+
+    We use the `ua` CLI with output formatted as JSON. This is sent
+    as-is and parsed by Landscape Server because the JSON content is
+    considered "Experimental" and we don't want to have to change in
+    both Client and Server in the event that the format changes.
+    """
+
+    run_interval = 15
```
Comment (silverdrake11) [inline landscape/client/monitor/ubuntuproinfo.py:18]: Should be 900 seconds

**Review by silverdrake11 — CHANGES_REQUESTED** [review]

File: landscape/client/monitor/ubuntuproinfo.py (hunk 1 above)
Comment (Perfect5th) [reply landscape/client/monitor/ubuntuproinfo.py]: oops - fixed.

**Review by silverdrake11 — APPROVED** [review]
LGTM +1

**Pull request merged**